openmc_models/inf_lat_dep/corewise/iter1/bcc_model.py

```diff
@@ -0,0 +1,114 @@
+import openmc
+import openmc.deplete
+import numpy as np
+
+
+#materials
+#graphite based on a3-3, triso layers pulled from reported values in
+#Neutronics characteristics of a 165 MWth Xe-100 reactor, Mulder et al
+
+#d_steps are (in days):
+#d_steps = [1] + [4] + [4] + [10]*9 + [25]*10 + [50]*24
+materials = openmc.Materials.from_xml('../materials.xml')
+dep_file = 'i0-dep-res.h5'
+res = openmc.deplete.Results(dep_file)
+dep_t = res.get_times()
+
+f = open('../nuc_list.txt', 'r')
+nuc_w_data = []
+for n in f.readlines():
+    nuc_w_data.append(n.strip())
+
+nuc_in_res = list(res[0].index_nuc.keys())
+
+nuclides = list(set(nuc_w_data) & set(nuc_in_res))
+
+print(nuclides)
+
+p_bins = [(0,19),(19,26),(26,31),(31,36),(36,41),(41,None)]
+p_comps = {'p01':{},
+           'p12':{},
+           'p23':{},
+           'p34':{},
+           'p45':{},
+           'p56':{}}
+p_t = [sum(dep_t[0:19]), 
+       sum(dep_t[19:26]), 
+       sum(dep_t[26:31]), 
+       sum(dep_t[31:36]),
+       sum(dep_t[36:41]), 
+       sum(dep_t[41:])]
```

Comment:
These ranges appear in a lot of places. Could you make a variable that holds these ranges once rather than repeating them through out the code?